packages/react-dom/src/__tests__/ReactTestUtils-test.js

```diff
@@ -9,11 +9,11 @@
 
 'use strict';
 
-let createRenderer;
-let React;
-let ReactDOM;
-let ReactDOMServer;
-let ReactTestUtils;
+import ReactTestRenderer from 'react-test-renderer/shallow';
```

Comment:
Okay this is what confused me. It should say `ReactShallowRenderer`.

Reply:
potato, potato... fixed :)